python/paddle/fluid/tests/unittests/mkldnn/test_matmul_mkldnn_op.py

```diff
@@ -519,43 +519,6 @@ def init_data_type(self):
         self.data_type_ = np.int8
 
 
-class TestMatMulOpTransposeReshapeTransposeAxisNotSupportedException(
```

Comment:
Why these tests are deleted?

Reply:
These tests only checked if after setting incorrect values for `fuse_reshape_out` and `fuse_transpose_out` test would fail. I deleted it, because the only way of setting these parameters is via fuse_pass, which has all PADDLE_ENFORCES to prevent such case. Also, we have separate unit test dedicated for `matmul+transpose+reshape` fuse pass.

Reply:
Really nice that you have spotted that! Thanks!